DatadogRUM/Sources/Integrations/WebViewEventReceiver.swift

```diff
@@ -39,23 +39,27 @@ internal final class WebViewEventReceiver: FeatureMessageReceiver {
         self.viewCache = viewCache
     }
 
+    /// Writes a Browser RUM event to the core.
+    ///
+    /// The receiver will inject current RUM context and apply server-time offset to the event.
+    ///
+    /// - Parameters:
+    ///   - message: The message containing the Browser RUM event.
+    ///   - core: The core to write the event.
     func receive(message: FeatureMessage, from core: DatadogCoreProtocol) -> Bool {
-        guard case let .webview(.rum(event)) = message else {
+        switch message {
+        case let .webview(.rum(event)):
+            receive(rum: event, from: core)
+        case let .webview(.telemetry(event)):
+            receive(telemetry: event, from: core)
```

Comment:
obsolete/ Starting from https://github.com/DataDog/dd-sdk-ios/pull/1744 we want to avoid passing the `core` reference around if not necessary and that includes message receiver APIs. We want to [delete the `core` parameter long term](https://github.com/DataDog/dd-sdk-ios/blob/723dc0d61198fad2c627fc2579d24bf991b015ef/DatadogInternal/Sources/MessageBus/FeatureMessageReceiver.swift#L29-L32). To move us closer to the goal, the https://github.com/DataDog/dd-sdk-ios/pull/1750 has cleaned up `WebViewEventReceiver`, but now we bring it back in this change. With availability of `featureScope` in this receiver, the `core` reference is obsolete in this file. Let's stick to one convention and goal 🙏.

Reply:
Sure, it was brought back while rebasing on develop, let me change that 👍